scripts/BuildLibsRelease.sh

```diff
@@ -1,113 +1,157 @@
-sudo mkdir -p /opt/local/lib
-sudo mkdir -p /opt/local/bin
+#!/bin/bash -e
 
-#setup permissions on /opt/local
-sudo chgrp -R staff /opt/local*
-sudo chmod -R g+w /opt/local*
+# System dependency checks...
+if [ ! -d /opt/local/lib ] || [ ! -d /opt/local/libdbg ] || [ ! -d /opt/local/bin ] || [ ! -d /opt/local/bin ] ; then
+  echo "/opt/local/bin, /opt/local/lib, /opt/local/libdbg and /opt/local/include must exist and be writable!"
```

Comment:
mild recommend for using `printf` over `echo`

here are some sample helpers that will do what you might expect
https://gist.github.com/DanInProgress/e444b7815745da4c1117f78dc9ebb2d3